proto/prysm/v1alpha1/beacon_block.proto

```diff
@@ -767,9 +767,9 @@ message SignedBuilderBidCapella {
 
 message BuilderBidDeneb {
     ethereum.engine.v1.ExecutionPayloadHeaderDeneb header = 1;
-    bytes value = 2 [(ethereum.eth.ext.ssz_size) = "32"];
```

Comment:
with this change I think client/builder/bid.go needs to be updated

Reply:
What change?

Reply:
I think this code
```
// Value --
func (b builderBidDeneb) Value() []byte {
	return b.p.Value
}

```

Reply:
Oh it's not removed, we just changed the ordering from 2 to 3

Reply:
oops i totally misread sorry